### ISSUE TITLE
Not able to install version 0.18.0

### ISSUE THREAD

**Issue by henri-hulski** [deschler/django-modeltranslation#637]
When trying to install the new version 0.18.0 I get the following error:
```
Collecting django-modeltranslation
  Using cached django-modeltranslation-0.18.0.tar.gz (70 kB)
  Installing build dependencies ... done
  Getting requirements to build wheel ... error
  error: subprocess-exited-with-error
  
  × Getting requirements to build wheel did not run successfully.
  │ exit code: 1
  ╰─> [66 lines of output]
      Traceback (most recent call last):
        File "/tmp/pip-build-env-684kgmvk/overlay/lib/python3.9/site-packages/setuptools/config/expand.py", line 85, in __getattr__
          return next(ast.literal_eval(value) for value in matching_values)
        File "/tmp/pip-build-env-684kgmvk/overlay/lib/python3.9/site-packages/setuptools/config/expand.py", line 85, in <genexpr>
          return next(ast.literal_eval(value) for value in matching_values)
        File "/usr/lib/python3.9/ast.py", line 105, in literal_eval
          return _convert(node_or_string)
        File "/usr/lib/python3.9/ast.py", line 104, in _convert
          return _convert_signed_num(node)
        File "/usr/lib/python3.9/ast.py", line 78, in _convert_signed_num
          return _convert_num(node)
        File "/usr/lib/python3.9/ast.py", line 69, in _convert_num
          _raise_malformed_node(node)
        File "/usr/lib/python3.9/ast.py", line 66, in _raise_malformed_node
          raise ValueError(f'malformed node or string: {node!r}')
      ValueError: malformed node or string: <ast.Call object at 0x7fd16d508f40>
      
      The above exception was the direct cause of the following exception:
      
      Traceback (most recent call last):
        File "/tmp/pip-build-env-684kgmvk/overlay/lib/python3.9/site-packages/setuptools/config/expand.py", line 191, in read_attr
          return getattr(StaticModule(module_name, spec), attr_name)
        File "/tmp/pip-build-env-684kgmvk/overlay/lib/python3.9/site-packages/setuptools/config/expand.py", line 87, in __getattr__
          raise AttributeError(f"{self.name} has no attribute {attr}") from e
      AttributeError: modeltranslation has no attribute __version__
      
      During handling of the above exception, another exception occurred:
      
      Traceback (most recent call last):
        File "/home/wadera/web/sancode/machinarium.shop/env/lib/python3.9/site-packages/pip/_vendor/pep517/in_process/_in_process.py", line 363, in <module>
          main()
        File "/home/wadera/web/sancode/machinarium.shop/env/lib/python3.9/site-packages/pip/_vendor/pep517/in_process/_in_process.py", line 345, in main
          json_out['return_val'] = hook(**hook_input['kwargs'])
        File "/home/wadera/web/sancode/machinarium.shop/env/lib/python3.9/site-packages/pip/_vendor/pep517/in_process/_in_process.py", line 130, in get_requires_for_build_wheel
          return hook(config_settings)
        File "/tmp/pip-build-env-684kgmvk/overlay/lib/python3.9/site-packages/setuptools/build_meta.py", line 177, in get_requires_for_build_wheel
          return self._get_build_requires(
        File "/tmp/pip-build-env-684kgmvk/overlay/lib/python3.9/site-packages/setuptools/build_meta.py", line 159, in _get_build_requires
          self.run_setup()
        File "/tmp/pip-build-env-684kgmvk/overlay/lib/python3.9/site-packages/setuptools/build_meta.py", line 281, in run_setup
          super(_BuildMetaLegacyBackend,
        File "/tmp/pip-build-env-684kgmvk/overlay/lib/python3.9/site-packages/setuptools/build_meta.py", line 174, in run_setup
          exec(compile(code, __file__, 'exec'), locals())
        File "setup.py", line 8, in <module>
          setup()
        File "/tmp/pip-build-env-684kgmvk/overlay/lib/python3.9/site-packages/setuptools/__init__.py", line 87, in setup
          return distutils.core.setup(**attrs)
        File "/tmp/pip-build-env-684kgmvk/overlay/lib/python3.9/site-packages/setuptools/_distutils/core.py", line 122, in setup
          dist.parse_config_files()
        File "/tmp/pip-build-env-684kgmvk/overlay/lib/python3.9/site-packages/setuptools/dist.py", line 860, in parse_config_files
          setupcfg.parse_configuration(
        File "/tmp/pip-build-env-684kgmvk/overlay/lib/python3.9/site-packages/setuptools/config/setupcfg.py", line 167, in parse_configuration
          meta.parse()
        File "/tmp/pip-build-env-684kgmvk/overlay/lib/python3.9/site-packages/setuptools/config/setupcfg.py", line 446, in parse
          section_parser_method(section_options)
        File "/tmp/pip-build-env-684kgmvk/overlay/lib/python3.9/site-packages/setuptools/config/setupcfg.py", line 417, in parse_section
          self[name] = value
        File "/tmp/pip-build-env-684kgmvk/overlay/lib/python3.9/site-packages/setuptools/config/setupcfg.py", line 238, in __setitem__
          value = parser(value)
        File "/tmp/pip-build-env-684kgmvk/overlay/lib/python3.9/site-packages/setuptools/config/setupcfg.py", line 552, in _parse_version
          return expand.version(self._parse_attr(value, self.package_dir, self.root_dir))
        File "/tmp/pip-build-env-684kgmvk/overlay/lib/python3.9/site-packages/setuptools/config/setupcfg.py", line 372, in _parse_attr
          return expand.read_attr(attr_desc, package_dir, root_dir)
        File "/tmp/pip-build-env-684kgmvk/overlay/lib/python3.9/site-packages/setuptools/config/expand.py", line 195, in read_attr
          return getattr(module, attr_name)
      AttributeError: module 'modeltranslation' has no attribute '__version__'
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: subprocess-exited-with-error

× Getting requirements to build wheel did not run successfully.
│ exit code: 1
╰─> See above for output.

note: This error originates from a subprocess, and is likely not a problem with pip.
```

**Comment by last-partizan** [comment]
Looking into this, thanks for report!

**Comment by last-partizan** [comment]
Should be fixed in 0.18.1
